src/frontend/apps/standalone_site/src/features/App/AppConfig.tsx

```diff
@@ -63,6 +63,8 @@ const AppConfig = ({ children }: PropsWithChildren<unknown>) => {
       login_html: config.login_html,
       footer_copyright: config.footer_copyright,
       vod_conversion_enabled: config.vod_conversion_enabled,
+      homepage_banner_title: config.homepage_banner_title,
+      homepage_banner_text: config.homepage_banner_text,
```

Comment:
Potentially, you could do that to increase the dependency:
```ts
setSiteConfig(config);
```
But `SiteConfig` must have the exact same type as `ConfigResponse`, for ts `Maybe` create a diff.
The best would have been that SiteConfig picks from ConfigResponse:
```ts
type SiteConfig = Pick<ConfigResponse, "is_default_site" | "logo_url" ...>
```
but I can see it is not easy, they are not in the same package.